### ISSUE TITLE
MegEngine support

### ISSUE THREAD

**Issue by lutzroeder** [lutzroeder/netron#607]
[shufflenet_v2_x1_0.tm.zip](https://github.com/lutzroeder/netron/files/9257733/shufflenet_v2_x1_0.tm.zip)
[mobilenetv1.tm.zip](https://github.com/lutzroeder/netron/files/9257882/mobilenetv1.tm.zip)

[shufflenet_v2_x1_0_state_dict.mge.zip](https://github.com/lutzroeder/netron/files/9286205/shufflenet_v2_x1_0_state_dict.mge.zip)
[shufflenet_v2_x1_0.mge.zip](https://github.com/lutzroeder/netron/files/9286206/shufflenet_v2_x1_0.mge.zip)
